### PR TITLE
add azure privatelink support

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

File: aiven/resource_azure_privatelink.go [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+			if err != nil {
+				return nil, "", err
+			}
+
+			log.Printf("[DEBUG] Got %s state while waiting for Azure privatelink to be active.", pl.State)
+
+			return pl, pl.State, nil
+		},
+		Delay:      10 * time.Second,
+		Timeout:    d.Timeout(schema.TimeoutCreate),
```
Comment (mhoffm-aiven) [inline aiven/resource_azure_privatelink.go:120]: could be `schema.TimeoutRead`

**Review by mhoffm-aiven — CHANGES_REQUESTED** [review]

File: aiven/resource_azure_privatelink.go (hunk 1 above)
Comment (ivan-savciuc) [reply aiven/resource_azure_privatelink.go]: done

**Comment by ivan-savciuc** [comment]
added deletion waiter

**Review by mhoffm-aiven — APPROVED** [review]
lgtm

**Pull request merged**